browser/ui/BUILD.gn

```diff
@@ -108,8 +109,14 @@ source_set("ui") {
     sources += [
       "webui/webcompat_reporter/webcompat_reporter_ui.cc",
       "webui/webcompat_reporter/webcompat_reporter_ui.h",
+      # brave_browser_commands are desktop only too
+      "webui/brave_browser_command/brave_browser_command_handler.cc",
```

Comment:
these should go in a BUILD.gn in `webui/brave_browser_command`. The chromium code predates splitting up browser/ui code

Reply:
if you run into any circular dep issues (I don't think there are any) you can leave it here for now and we'll follow-up

Reply:
Fixed in https://github.com/brave/brave-core/pull/26154/commits/661885a3a239d1be845516795e546c6903c96703